ext/lib/crypto/mbedtls/Kconfig

```diff
@@ -17,6 +17,14 @@
 #
 
 
+config MBEDTLS_EXTERNAL_CONFIG
+	bool
+	help
+	  This internal option controls if mbedTLS is configured by external
```

Comment:
You can use the term out-of-tree instead of 'external' to be more precise and to use the same terminology as elsewhere.

Reply:
Yes, I can use this term.